model-optimizer/extensions/ops/interpolate.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, graph: Graph, attrs: dict):
             'in_ports_count': 2,
             'out_ports_count': 1,
         }
+        if 'axes' in attrs and attrs['axes'] is not None:
+            attrs['axes'] = int64_array(attrs['axes'])
```

Comment:
Why do we need this change? Is there any place in the MO where the `axes` attribute is initialized incorrectly? Should we have an assert here that the `int64_array(attrs['axes'])` == np.array(attrs['axes'])?